src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -793,6 +793,9 @@ export default class DefaultAudioVideoController
     }
     this.sessionStateController.perform(SessionStateControllerAction.FinishDisconnecting, () => {
       if (!reconnecting) {
+        // Do a hard reset of the signaling client in case this controller is reused;
+        // this will also can `this.meetingSessionContext` to be reset if reused.
+        this.meetingSessionContext.signalingClient = null;
```

Comment:
Do you need to reset it here as well? https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideocontroller/DefaultAudioVideoController.ts#L736

Reply:
Sure, I'm not super familiar with that path but I think it makes sense to reset it there too (or at least it doesn't hurt).